Cargo.lock

```diff
@@ -8559,9 +8559,9 @@ dependencies = [
 
 [[package]]
 name = "pallet-message-queue"
-version = "39.0.0"
+version = "39.0.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "20e65a37881d1998546254a5e50a1f768b3f82deabe774e750f4ea95aba8030c"
+checksum = "8bb779dbfd58b3a4e6a84245392fe9d8c31a746ff7c6db414e91c012663859c3"
```

Comment:
To pick up this fix for other runtimes we need to bump them too. Not sure if the fix is urgent, if the other system chains have any overweight stuck messages in the queue.
But if they don't, I don't personally think it's worth the runtime upgrade, and we can go without bumping them.

Reply:
Good Q. I think it should be fine to only bump PAH. Last time we checked it was only Hydra affected (Astar was false positive).